src/core/thread/network_data_service.hpp

```diff
@@ -150,63 +153,221 @@ class BackboneRouter
 #endif // #if (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2)
 
 /**
- * This type implements Thread Network Data "SRP Server Service" server data generation and parsing.
+ * This type implements Thread Network Data "DNS/SRP Service Anycast Address" generation and parsing.
  *
  */
-class SrpServer
+class DnsSrpAnycast
 {
 public:
     enum : uint8_t
     {
-        kServiceNumber = OPENTHREAD_CONFIG_SRP_SERVER_SERVICE_NUMBER, ///< SRP Sever Service number
+        kServiceNumber = 0x02, ///< The service number of a `DnsSrpAnycast` entry.
```

Comment:
Should we define both `kServiceNumber` values in the same header file? class `DnsSrpAnycast` uses `OPENTHREAD_CONFIG_SRP_SERVER_SERVICE_NUMBER ` which is defined in `srp_server.h`.

Reply:
I think once we define the models in spec (and decide/specify the service numbers), we should remove the config `OPENTHREAD_CONFIG_SRP_SERVER_SERVICE_NUMBER` fully.

Reply:
I will remove this config option in next push.